build.gradle

```diff
@@ -240,12 +241,13 @@ allprojects { subproj ->
 }
 
 subprojects { subproj ->
+	apply plugin: 'propdeps'
+	apply plugin: 'propdeps-maven'
+	apply plugin: 'propdeps-idea'
+	apply plugin: 'propdeps-eclipse'
+
```

Comment:
Why did you move all them here? At least `'propdeps-maven'` could/should stay in the "is Maven-based project" closure. Not?

Reply:
I moved them here for a couple of reasons:

1. I needed to be able to use `optional()` further down where I declare `optional("com.google.errorprone:error_prone_annotations:${errorproneVersion}")`.
2. I was simply unaware that `'propdeps-maven'` could be declared separately. :)

I'll move `'propdeps-maven'` back later today and see how it impacts the error-prone side of the build.

Reply:
There's no observable impact on error-prone for moving `'propdeps-maven'` back on my end, so let's see what Travis/Appveyor have to say.